doc/changes/latest.rst

```diff
@@ -13,126 +15,152 @@ HIGHLIGHTS
  | **Support for Python 3.8 has been dropped.**
  | **We recommend upgrading to Python 3.11 or above.**
  |
+ | **This version is the first one to support Python 3.13.**
+ |
  | **Minimum supported versions of the following packages have been bumped up:**
  | - numpy -- 1.22.4
  | - nibabel -- 5.2.0
  | - scikit-learn -- 1.4.0
  | - joblib -- 1.2.0
  | - pandas -- 2.2.0
 
-NEW
----
+-   This version includes a new "surface" API that allow to represent surface meshes and data.
+
+-   Single hemisphere meshes can be instantiated with
+    :class:`~nilearn.surface.SurfaceMesh`, :class:`~nilearn.surface.InMemoryMesh` or :class:`~nilearn.surface.FileMesh`.
+    :class:`~nilearn.surface.SurfaceMesh`
+
+-   :class:`~nilearn.surface.PolyMesh` (and :class:`~nilearn.surface.PolyData`)
+    can be used to instantiate mesh (and respectively data) objects with both hemisphere.
+
+-   :class:`~nilearn.surface.SurfaceImage` are objects that hold meshes and data for both hemispheres.
+
+-   :class:`~nilearn.surface.SurfaceImage`, :class:`~nilearn.surface.PolyMesh`, :class:`~nilearn.surface.PolyData`
+    allow you to load and save mesh or data from one hemisphere from or to a Gifti file.
+    :class:`~nilearn.surface.SurfaceImage` can also be instantiated directly with a nifti image and a mesh to project it on.
+
+-   Surface plotting functions have been adapted to work with :class:`~nilearn.surface.SurfaceImage` object.
+
+-   To facilitate extracting data from those :class:`~nilearn.surface.SurfaceImage` objects,
+    this new release also introduces :class:`~nilearn.maskers.SurfaceMasker` and :class:`~nilearn.maskers.SurfaceLabelsMasker`
+    as counterparts to the volumetric :class:`~nilearn.maskers.NiftiMasker` and :class:`~nilearn.maskers.NiftiLabelsMasker`.
+
+-   Several classes will already accept :class:`~nilearn.surface.SurfaceImage` and :class:`~nilearn.maskers.SurfaceMasker` as inputs:
+    :class:`~nilearn.decoding.Decoder`, :class:`~nilearn.decoding.DecoderRegressor`,
+    :class:`~nilearn.decoding.FREMClassifier`, :class:`~nilearn.decoding.FREMRegressor`,
+    :class:`~nilearn.glm.first_level.FirstLevelModel`, :class:`~nilearn.glm.second_level.SecondLevelModel`.
+    Moreover :func:`~nilearn.glm.first_level.first_level_from_bids` will be able to to build :class:`~nilearn.glm.first_level.FirstLevelModel`
+    with the ``fsaverage5`` files generated by fmriprep to directly analysis with surface data.
+
 
```

Comment:
Should we mention whats NOT included yet and that people can expect them in future releases?
For example: 
1. Parcellations and DictLearning object do not support SurfaceImages
2. No SurfaceMapsMasker yet
3. No Multi* markers yet

Reply:
I would say let's not mention it because the list could grow very long: we have a lot of features that are not implemented: surface smoothing, utility functions to more easily work with surface images... so let's stick to things that are.